### DISCUSSION TITLE
在powershell运行启动命令的时候提示由于目标计算机积极拒绝，无法连接。')))有人碰到这情况吗

### DISCUSSION THREAD

**Discussion by 965797199** [GaiZhenbiao/ChuanhuChatGPT#519] [Q&A] [answered]
/
![image](https://user-images.githubusercontent.com/68554292/229391881-9b75c6e5-cbf2-482a-8100-7951243a7623.png)
如图，我查了下说是要关代理，我也关过了还是不行

**Comment by huajiyu2008** [comment]
遇见同样问题，关闭代理后还是运行不了。

**Comment by xiaoyidon** [comment]
同样的问题，昨天的版本还能正常运行，今天更新后不行了

**Comment by Handrail4015** [comment]
遇见同样问题+1

**Comment by GaiZhenbiao** [comment]
应该是没有代理运行在这个端口上吧

**Comment by Keldos-Li** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/23137268/230115615-7475db01-359d-4f98-9536-492bcdddefd0.png)
不直接设置系统代理的，复制config-example.json到自己的config.json时把proxy相关的这两行删掉，example中的值只是示例

**Reply by 965797199** [reply]
大佬，改完后能进去了，但是又有新问题了，想问问你知不知道这个情况怎么解决
![115](https://user-images.githubusercontent.com/68554292/230142704-c16334b4-0512-4121-b346-c0010483ad03.png)
![1157](https://user-images.githubusercontent.com/68554292/230142709-eff58eae-7d9a-4c65-8533-7983644b8cc9.png)

**Reply by Keldos-Li** [reply]
https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98#something-went-wrong-expecting-value-line-1-column-1-char-0

**Reply by 965797199** [reply]
![1234](https://user-images.githubusercontent.com/68554292/230292284-382a358d-b7d7-44a0-8250-47303399d4a8.png)
大佬这问题有点多。。这个我用gpt问过了，并且也在你给的那个链接里找过了没看到这个问题，gpt上说这个错误提示表明您的环境中没有安装“llama_index”模块。但是我已经安装了，并且那个安装命运也运行过并且加载完了，还是不行

**Reply by Keldos-Li** [reply]
<img width="1172" alt="image" src="https://user-images.githubusercontent.com/23137268/230299073-8ce96df1-77fa-4263-a470-0f3cb58d6065.png">
怎么会没有这个问题呢，不是有吗。
这些问题都是一样的……要更新依赖……

你直接在文件目录中执行
```
pip install -r requirements.txt --upgrade
```
吧

**Reply by 965797199** [reply]
我在回复之前就更新过了，还是错误才来问的![333](https://user-images.githubusercontent.com/68554292/230306034-1af9a65a-30f1-49d0-b423-eb45af040a10.png)

**Comment by xiaoyidon** [comment]
感谢解答，确实是代理的问题，多谢








------------------ 原始邮件 ------------------
发件人: Keldos ***@***.***&gt;
发送时间: 2023年4月5日 22:41
收件人: GaiZhenbiao/ChuanhuChatGPT ***@***.***&gt;
抄送: xiaoyidon ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [GaiZhenbiao/ChuanhuChatGPT] 在powershell运行启动命令的时候提示由于目标计算机积极拒绝，无法连接。')))有人碰到这情况吗 (Discussion #519)





 

 不直接设置系统代理的，复制config-example.json到自己的config.json时把这两删掉

**Comment by xiaoyidon** [comment]
https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98%23something-went-wrong-expecting-value-line-1-column-1-char-0






------------------ 原始邮件 ------------------
发件人: 965797199 ***@***.***&gt;
发送时间: 2023年4月6日 00:21
收件人: GaiZhenbiao/ChuanhuChatGPT ***@***.***&gt;
抄送: xiaoyidon ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [GaiZhenbiao/ChuanhuChatGPT] 在powershell运行启动命令的时候提示由于目标计算机积极拒绝，无法连接。')))有人碰到这情况吗 (Discussion #519)





 
大佬，改完后能进去了，但是又有新问题了，想问问你知不知道这个情况怎么解决
